wireprotocol/wireprimitives.go

```diff
@@ -117,6 +117,8 @@ func WriteSchema(schema types.Schema) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+// Note this expects the token to be consumed prior to execution
```

Comment:
comments should star with `// ReadSchema ...`
also, I don't quite understand the comment.

Reply:
i simply produced a buffer with WriteSchema and then consumed that same buffer with ReadSchema and it failed and I was just attempting to communicate that is not expected behavior